src/main/java/com/jmb/springfactory/model/factory/RolSamples.java

```diff
@@ -0,0 +1,15 @@
+package com.jmb.springfactory.model.factory;
+
+public final class RolSamples {
+
+  private RolSamples() {}
+  
+  public final static String ID_ROL_TEST_1 = "1";
+  public final static String NAME_ROL_TEST_1 = "Admin";
+  
+  public final static String ID_ROL_TEST_2 = "2";
+  public final static String NAME_ROL_TEST_2 = "Operator";
+
+  public final static String ID_ROL_TEST_3 = "3";
+  public final static String NAME_ROL_TEST_3 = "Engineer";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AModifiersOrderCheck)